### PR TITLE
node: Governor token list update

### PR DESCRIPTION
**Pull request by djb15**
```
Tokens before = 1524
Tokens after = 1534

Tokens added = 13:
<WH_chain_id>-<WH_token_addr>-<token_symbol>

[
 "1-6cccc33e6627899bd00d679cca0b56d788a1861b8de1ea3691e870710b92447c-LOCKIN",
 "4-0000000000000000000000008db1d28ee0d822367af8d220c0dc7cb6fe9dc442-ETHPAD",
 "4-000000000000000000000000f585b5b4f22816baf7629aea55b701662630397b-VOW",
 "10-0000000000000000000000002f733095b80a04b38b0d10cc884524a3d09b836a-USDC.e",
 "15-9bef49ce5caa641ff4455e44855fe85a80fb71aecf70b59fc54164b309304e66-USDC",
 "22-b848656c196bccfa7b9708992b6af7714afce6a56e4689da5aed63b14e99757b-WETH",
 "22-93601512902fe46ad6c51440c23a1a7e166c4b60e773579eda639c113a81325a-USDT",
 "23-0000000000000000000000001b896893dfc86bb67cf57767298b9073d2c1ba2c-Cake",
 "23-000000000000000000000000fa7f8980b0f1e64a2062791cc3b0871572f1f7f0-UNI",
 "23-00000000000000000000000025d887ce7a35172c62febfd67a1856f20faebb00-PEPE",
 "30-0000000000000000000000003055913c90fcc1a6ce9a358911721eeb942013a1-Cake",
 "30-000000000000000000000000858c50c3af1913b0e849afdb74617388a1a5340d-SQT",
 "30-00000000000000000000000064b88c73a5dfa78d1713fe1b4c69a22d7e0faaa7-MAV"
]

Tokens removed = 3:
<WH_chain_id>-<WH_token_addr>-<token_symbol>

[
 "2-000000000000000000000000bf5495efe5db9ce00f80364c8b423567e58d2110-ezETH",
 "5-0000000000000000000000008f006d1e1d9dc6c98996f50a4c810f17a47fbf19-NSFW",
 "24-0000000000000000000000006c84a8f1c29108f47a79964b5fe888d4f4d0de40-tBTC"
]

Tokens with changed symbols = 1:
<WH_chain_id>-<WH_token_addr>-<old_token_symbol>-><new_token_symbol>

[
 "23-000000000000000000000000fa5ed56a203466cbbc2430a43c66b9d8723528e7-agEUR->EURA"
]

Tokens with invalid symbols = 0:
<WH_chain_id>-<WH_token_addr>-<token_symbol>

[]

Tokens with significant price drops (>50%) = 45:

[
 {
  "token": "1-204cc66e5cc71f4795cfafe7b922d9e9bfe57b6ae41fb5ab6d9e65a8845ee90f-BILLY",
  "previousPrice": 0.04530248,
  "newPrice": 0.01775129,
  "percentageChange": "-60.8"
 },
 {
  "token": "1-8fa66a1378bce9e01b9496615ed38c132c586fa36e29f6b38eace0bf7b3d4b69-ca",
  "previousPrice": 0.00007541,
  "newPrice": 0.00003744,
  "percentageChange": "-50.4"
 },
 {
  "token": "1-eac40dce1cfccb5ca30a878aebb3025763a8d3f0c4f15c3e858b98899982b71f-KSG",
  "previousPrice": 0.00090957,
  "newPrice": 0.00033955,
  "percentageChange": "-62.7"
 },
 {
  "token": "1-6b054395432c64d8250648371121f271fc3b8e36621384b9e91956bf02307c13-DOKI",
  "previousPrice": 0.00056559,
  "newPrice": 0.00027244,
  "percentageChange": "-51.8"
 },
 {
  "token": "1-8f8eda6d92c4afa64e25a02b8cc6e9f4026cf3007f48f279d842594d0328e121-WGC",
  "previousPrice": 3.41711e-7,
  "newPrice": 1.58686e-7,
  "percentageChange": "-53.6"
 },
 {
  "token": "1-e66b5c6e86ec048a73a370e0588e1dcbaa16f532b8e5790eb830386b3ae947fc-",
  "previousPrice": 0.00091972,
  "newPrice": 0.00018487,
  "percentageChange": "-79.9"
 },
 {
  "token": "1-11615a73ca119d91ad275cf19659f206265356fbd4161175b6c2cee43ceb267a-STATE",
  "previousPrice": 4.338e-9,
  "newPrice": 4.261e-12,
  "percentageChange": "-99.9"
 },
 {
  "token": "1-072844c94ab6a7097e1134511934be65304b482d6e8850493ce05ac78755dbea-UWU",
  "previousPrice": 0.00109061,
  "newPrice": 0.00051776,
  "percentageChange": "-52.5"
 },
 {
  "token": "1-e350770814014371e2277d39b550570dd02e77db5f266e58f68f023c277f14af-ily",
  "previousPrice": 0.0019989,
  "newPrice": 0.00073558,
  "percentageChange": "-63.2"
 },
 {
  "token": "1-05beb884b642b260d9c2b53cb3d876b8b4de72aea0d787d779bf4266688a3935-",
  "previousPrice": 0.01108076,
  "newPrice": 0.00408788,
  "percentageChange": "-63.1"
 },
 {
  "token": "1-e3669aa0deb30991efc974beb9afd905b02d4ee2eb930e627358464d1c9e6433-WBS",
  "previousPrice": 0.00058359,
  "newPrice": 0.00012287,
  "percentageChange": "-78.9"
 },
 {
  "token": "1-4262f09434e938b2714803d3aa425d1d543a735ba324109087760d83deae653d-SPWN",
  "previousPrice": 0.00002693,
  "newPrice": 0.0000106,
  "percentageChange": "-60.6"
 },
 {
  "token": "2-000000000000000000000000725c263e32c72ddc3a19bea12c5a0479a81ee688-BMI",
  "previousPrice": 0.00631235,
  "newPrice": 0.00309745,
  "percentageChange": "-50.9"
 },
 {
  "token": "2-00000000000000000000000065ef703f5594d2573eb71aaf55bc0cb548492df4-MULTI",
  "previousPrice": 0.412999,
  "newPrice": 0.195538,
  "percentageChange": "-52.7"
 },
 {
  "token": "2-0000000000000000000000001f7e5118521b550bb1a9b435727c003eb033fc51-AGLA",
  "previousPrice": 0.02502795,
  "newPrice": 0.00819791,
  "percentageChange": "-67.2"
 },
 {
  "token": "2-00000000000000000000000000e679ba63b509182c349f5614f0a07cdd0ce0c5-DAMEX",
  "previousPrice": 0.01902351,
  "newPrice": 0.00772967,
  "percentageChange": "-59.4"
 },
 {
  "token": "2-00000000000000000000000087edffde3e14c7a66c9b9724747a1c5696b742e6-SWAG",
  "previousPrice": 0.00695009,
  "newPrice": 0.00216092,
  "percentageChange": "-68.9"
 },
 {
  "token": "2-00000000000000000000000008389495d7456e1951ddf7c3a1314a4bfb646d8b-CRPT",
  "previousPrice": 0.04159604,
  "newPrice": 0.01547705,
  "percentageChange": "-62.8"
 },
 {
  "token": "2-000000000000000000000000c4c2614e694cf534d407ee49f8e44d125e4681c4-CHAIN",
  "previousPrice": 0.0475276,
  "newPrice": 0.01980198,
  "percentageChange": "-58.3"
 },
 {
  "token": "2-00000000000000000000000078a0a62fba6fb21a83fe8a3433d44c73a4017a6f-OX",
  "previousPrice": 0.00080818,
  "newPrice": 0.0002877,
  "percentageChange": "-64.4"
 },
 {
  "token": "2-000000000000000000000000cb84d72e61e383767c4dfeb2d8ff7f4fb89abc6e-VEGA",
  "previousPrice": 0.191197,
  "newPrice": 0.056596,
  "percentageChange": "-70.4"
 },
 {
  "token": "2-000000000000000000000000e516d78d784c77d479977be58905b3f2b1111126-SPWN",
  "previousPrice": 0.00002693,
  "newPrice": 0.0000106,
  "percentageChange": "-60.6"
 },
 {
  "token": "2-000000000000000000000000f64265e65c4529879a7abf467e00d39e39c0b0da-RFKJ",
  "previousPrice": 0.00001654,
  "newPrice": 0.00000588,
  "percentageChange": "-64.4"
 },
 {
  "token": "2-0000000000000000000000003301ee63fb29f863f2333bd4466acb46cd8323e6-AKITA",
  "previousPrice": 1.27373e-7,
  "newPrice": 5.6747e-8,
  "percentageChange": "-55.4"
 },
 {
  "token": "2-000000000000000000000000beef01060047522408756e0000a90ce195a70000-APTR",
  "previousPrice": 0.03537705,
  "newPrice": 0.00851258,
  "percentageChange": "-75.9"
 },
 {
  "token": "2-000000000000000000000000a3a80559bbc1989654e6b8385f5001fa1333332f-RZR",
  "previousPrice": 0.101606,
  "newPrice": 0.01277724,
  "percentageChange": "-87.4"
 },
 {
  "token": "2-000000000000000000000000a462bde22d98335e18a21555b6752db93a937cff-BOBBY",
  "previousPrice": 0.00001654,
  "newPrice": 0.00000588,
  "percentageChange": "-64.4"
 },
 {
  "token": "4-00000000000000000000000042c95788f791a2be3584446854c8d9bb01be88a9-HBR",
  "previousPrice": 0.080642,
  "newPrice": 0.03326192,
  "percentageChange": "-58.8"
 },
 {
  "token": "4-000000000000000000000000552594612f935441c01c6854edf111f343c1ca07-GWT",
  "previousPrice": 0.00091972,
  "newPrice": 0.00018487,
  "percentageChange": "-79.9"
 },
 {
  "token": "4-000000000000000000000000c342774492b54ce5f8ac662113ed702fc1b34972-BGEO",
  "previousPrice": 0.00175404,
  "newPrice": 0.00014432,
  "percentageChange": "-91.8"
 },
 {
  "token": "4-0000000000000000000000000a356f512f6fce740111ee04ab1699017a908680-UFARM",
  "previousPrice": 0.00033887,
  "newPrice": 0.00013391,
  "percentageChange": "-60.5"
 },
 {
  "token": "4-000000000000000000000000a75d9ca2a0a1d547409d82e1b06618ec284a2ced-WMX",
  "previousPrice": 0.00129969,
  "newPrice": 0.00026762,
  "percentageChange": "-79.4"
 },
 {
  "token": "4-00000000000000000000000083adb07bb91ddde95a24982f1b2d343963ba3995-Fwcl",
  "previousPrice": 0.08117,
  "newPrice": 0.02347393,
  "percentageChange": "-71.1"
 },
 {
  "token": "4-0000000000000000000000002c717059b366714d267039af8f59125cadce6d8c-MHUNT",
  "previousPrice": 0.00237551,
  "newPrice": 0.00050053,
  "percentageChange": "-78.9"
 },
 {
  "token": "4-0000000000000000000000009fb9a33956351cf4fa040f65a13b835a3c8764e3-MULTI",
  "previousPrice": 0.412999,
  "newPrice": 0.195538,
  "percentageChange": "-52.7"
 },
 {
  "token": "4-000000000000000000000000fdfd27ae39cebefdbaac8615f18aa68ddd0f15f5-GHD",
  "previousPrice": 0.00001,
  "newPrice": 0.00000136,
  "percentageChange": "-86.4"
 },
 {
  "token": "4-00000000000000000000000079ebc9a2ce02277a4b5b3a768b1c0a4ed75bd936-CATGIRL",
  "previousPrice": 3.82347e-10,
  "newPrice": 6.108e-12,
  "percentageChange": "-98.4"
 },
 {
  "token": "4-0000000000000000000000002d94172436d869c1e3c094bead272508fab0d9e3-RCG",
  "previousPrice": 0.01457408,
  "newPrice": 0.00454774,
  "percentageChange": "-68.8"
 },
 {
  "token": "5-00000000000000000000000078a0a62fba6fb21a83fe8a3433d44c73a4017a6f-OX",
  "previousPrice": 0.00080818,
  "newPrice": 0.0002877,
  "percentageChange": "-64.4"
 },
 {
  "token": "5-00000000000000000000000081382e9693de2afc33f69b70a6c12ca9b3a73f47-DOSE",
  "previousPrice": 0.00022112,
  "newPrice": 0.00010202,
  "percentageChange": "-53.9"
 },
 {
  "token": "5-000000000000000000000000011734f6ed20e8d011d85cf7894814b897420acf-ACRE",
  "previousPrice": 0.00008705,
  "newPrice": 0.00004087,
  "percentageChange": "-53.0"
 },
 {
  "token": "10-0000000000000000000000009fb9a33956351cf4fa040f65a13b835a3c8764e3-MULTI",
  "previousPrice": 0.412999,
  "newPrice": 0.195538,
  "percentageChange": "-52.7"
 },
 {
  "token": "23-000000000000000000000000aa54e84a3e6e5a80288d2c2f8e36ea5ca3a3ca30-$SHARBI",
  "previousPrice": 0.00000547,
  "newPrice": 5.31839e-7,
  "percentageChange": "-90.3"
 },
 {
  "token": "30-00000000000000000000000054a65b16d18527160fafc6fcfebaed47c2976dfa-RFKJ",
  "previousPrice": 0.00001654,
  "newPrice": 0.00000588,
  "percentageChange": "-64.4"
 }
]
```

### CODE REVIEW

File: node/hack/governor/src/index.ts [hunk 1]
```diff
@@ -246,7 +246,7 @@ axios
       // The token symbol has changed
       // We take a substring of the symbol to cut the "=" character we added above
       else if (tokenParts[0] + "-" + tokenParts[1] + "-" + newTokenSymbol.substring(1) != token) {
-        changedSymbols.push(token + "->" + newTokenSymbol);
+        changedSymbols.push(token + "->" + newTokenSymbol.substring(1));
```
Comment (johnsaigle) [inline node/hack/governor/src/index.ts:249]: Are the changes to this script supposed to be in  this PR?

File: node/pkg/governor/manual_tokens.go [hunk 2]
```diff
@@ -11,7 +11,6 @@ func manualTokenList() []tokenConfigEntry {
 		{chain: 13, addr: "0000000000000000000000005fff3a6c16c2208103f318f4713d4d90601a7313", symbol: "KLEVA", coinGeckoId: "kleva", decimals: 18, price: 0.086661},
 		{chain: 13, addr: "0000000000000000000000005096db80b21ef45230c9e423c373f1fc9c0198dd", symbol: "WEMIX", coinGeckoId: "wemix-token", decimals: 18, price: 1.74},
 		{chain: 15, addr: "0000000000000000000000000000000000000000000000000000000000000000", symbol: "NEAR", coinGeckoId: "near", decimals: 24, price: 4.34},
-		{chain: 22, addr: "93601512902fe46ad6c51440c23a1a7e166c4b60e773579eda639c113a81325a", symbol: "USDT", coinGeckoId: "tether", decimals: 6, price: 1.00},
```
Comment (johnsaigle) [inline node/pkg/governor/manual_tokens.go:14]: Confused about this one too. This change would've been done by you rather than the script right?

Comment (djb15) [reply node/pkg/governor/manual_tokens.go:14]: Yep I've batched in a couple of changes manually here. The github action fails when the test fails and the easiest way to get the changes in is to do it manually

File: node/hack/governor/src/index.ts (hunk 1 above)
Comment (djb15) [reply node/hack/governor/src/index.ts:249]: Yep I noticed the output was missing (and had the extra `=` char) so fixed that as part of this PR

**Review by johnsaigle — APPROVED** [review]

**Review by Mikhai56 — APPROVED** [review]

**Review by SEJeff — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Pull request merged**